21th Small Improvments

zumbak04 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Allowed to combine the warrior and necromancer classes.
**How to test:** start as Alexei Barov in the 603 bookmark, make sure you see **Learn Necromancy** in the decision list.
- If you start as Cairne (or any character that should start with Infertile trait), you get Infertile trait silently (without event firing).
- Hakkari and other not-kind religions don't get a penalty for dueling priests.
- N'raqi and C'Thraxxi no longer have -10 opinion towards each other.
- C'Thraxxi have +10 Shath'gral opinion since they're worshipped as demi-gods.
- Nerfed Dryad, Frost Nymph, Ancient races: they have +5 Natural opinion now.
- Hive government allows enatic and enatic-cognatic gender law and tanistry succession.
- Your courtiers and barons get Magical Withdrawal if you refuse to find a new source of magic.
- Buying mana crystals no longer cost a lot of gold, but give tax penalty.
- If the Sunwell is corrupted by Fel or Void, non-Fel/non-Void elves feel magical addiction.
- Magical Withdrawal takes less health.
- Added random events around magical addiction: an event where you can kill a character and steal their mana crystals, an event where you can get magical addiction immunity.

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Developer changelog:
- Yowler, the ruler of the Redridge gnolls in the 603 bookmark, got a father, Yowler.
- Added a clean-up event that clears Magical Withdrawal in case you convert to Fel/Void religion having Magical Withdrawal.

## Script changelog (ignore if you can't read CK2 script):
- `WCRAC.400` is fired with a delay.
- Moved addiction events to `wc_addiction_events.txt`.

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
- `add_rival <Character ID>` to make the character your rival and be able to duel.
- Play a high elf in the 605 bookmark, `event WCADD.1050` to feel magical addiction.
- Play a high elf in the 605 bookmark, `event WCADD.1100` to steal mana crystals.
- Play a high elf in the 605 bookmark, `event WCADD.1150` to get immunity.
- `add_trait has_magical_withdrawal` to get Magical Withdrawal.

Closes #1387, closes #1563, closes #682, closes #765, closes #1641, closes #1655, closes #1638.

arithon commented on localisation/000_wc_traits.csv:
```diff
@@ -357,7 +357,7 @@ eyes_mutation;Multiple Eyes;Yeux multiples;Mehrere Augen;;;;;;;;;;;x
 creature_nraqi;N'raqi;N'raqi;N'raqi;;;;;;;;;;;x
 creature_nraqi_desc;This character is n'raqi. N'raqi, of the faceless ones, are a monstrous race that arose from the organic matter that seeped from the Old Gods' blighted forms during the primordial era of Azeroth. N'raqi are very insidious and cruel, their uncanny forms, resembling something between a mollusk and an insect, can plunge into shock every mortal creature.;;Dieser Charakter ist ein  ...
-creature_cthraxxi;C'thraxxi;C'thraxxi;C'thraxxi;;;;;;;;;;;x
+creature_cthraxxi;C'Thrax;C'thraxxi;C'thraxxi;;;;;;;;;;;x
```
can you change it in all 3 languages pls?

zumbak04 replied:
Done in 4746f97ff9c43457f23703dd05f9ac36153ddb04.

zumbak04 commented:
> * Steal mana crystals event doesn’t remove Magical Withdrawal from courtiers and barony vassals.

Changed in 6d4fddd81070e92e17c95d44b4ebc33c20e1a2a8.

> One more thing - it seems the Illidan chain is broken. His war against Scourge ends as soon as Arthas becomes the Lich King in 605 bookmark.

Fixed on another branch, in 0ca4e0d9905ae095864be3a4fde65ba6408a4d66.
